cmd/neofs-cli/internal/key/raw.go

```diff
@@ -16,13 +16,8 @@ import (
 
 var errCantGenerateKey = errors.New("can't generate new private key")
 
-// Get returns private key from the following sources:
-// 1. WIF
-// 2. Raw binary key
```

Comment:
Raw binary still may be usefull. We want to prevent key leak via WIF copy-paste in shell and similar insecure scenarios first.

Reply:
updated.